src/tests/ftest/osa/osa_dmg_negative_test.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from osa_utils import OSAUtils
+from test_utils_pool import TestPool
+from apricot import skipForTicket
+
+
+class OSADmgNegativeTest(OSAUtils):
+    # pylint: disable=too-many-ancestors
+    """
+    Test Class Description: This test
+    passes invalid parameters to dmg OSA
+    related commands and check whether
+    appropriate errors are returned.
+
+    :avocado: recursive
+    """
+    def setUp(self):
+        """Set up for test case."""
+        super(OSADmgNegativeTest, self).setUp()
```

Comment:
(pylint-super-with-arguments) Consider using Python 3 style super() without arguments

Reply:
python3 issues will not be fixed now.